app/views/posts/_post_card.html.erb

```diff
@@ -0,0 +1,8 @@
+<article class="post border">
+    <h3 class="title"><%= post.title %></h3>
+    <p><%= post.text %></p>
+    <div class="counter">
+        <h5>Comments: <%= post.comments_counter %></h5>
+        <h5>Likes: <%= post.likes_counter %></h5>
+    </div>
+</article>
```

Comment:
- [ ] From the project requirements, clicking the post will redirect the user to the post, however, I can see that there is no link there, please add that in order to fulfill the project requirements 😃